dockerfiles/contracts-ci-linux/Dockerfile

```
@@ -49,8 +49,10 @@ RUN set -eux; \
 	cargo install cargo-dylint dylint-link && \
 	cargo install cargo-contract && \
 # download the latest `substrate-contracts-node` binary
-	curl -L "https://gitlab.parity.io/parity/substrate-contracts-node/-/jobs/artifacts/master/raw/artifacts/substrate-contracts-node-linux/substrate-contracts-node?job=build-linux" \
-		-o /usr/local/cargo/bin/substrate-contracts-node && \
+    curl -L -o substrate-contracts-node.zip 'https://gitlab.parity.io/parity/mirrors/substrate-contracts-node/-/jobs/1685841/artifacts/download?job=bud-linux' && \
```

```suggestion
    curl -L -o substrate-contracts-node.zip 'https://gitlab.parity.io/parity/mirrors/substrate-contracts-node/-/jobs/artifacts/main/download?job=build-linux' && \
```
still need to fix the link :)